src/main/java/com/terraformersmc/cinderscapes/block/BrambleBerryBushBlock.java

```diff
@@ -28,6 +27,10 @@
  * @project Cinderscapes
  */
 public class BrambleBerryBushBlock extends SweetBerryBushBlock {
+    private static final VoxelShape SMALL_SHAPE = Block.createCuboidShape(3.0D, 0.0D, 3.0D, 13.0D, 8.0D, 13.0D);
+    private static final VoxelShape MEDIUM_SHAPE = Block.createCuboidShape(3.0D, 0.0D, 3.0D, 13.0D, 13.0D, 13.0D);
+    private static final VoxelShape LARGE_SHAPE = Block.createCuboidShape(2.0D, 0.0D, 2.0D, 14.0D, 14.0D, 14.0D);
```

Comment:
Are these significantly different than the models for the sweet berry bush, could you kind of describe visually what this change does? I don't know if it's entirely necessary.

Reply:
Firstly, the sweet berry bush uses a full block shape for the largest shape. In the case of the bramble berries, the model itself does not stretch that far.